aws/resource_aws_service_discovery_private_dns_namespace.go

```diff
@@ -1,6 +1,8 @@
 package aws
 
 import (
+	"crypto/sha1"
```

Comment:
We actually do not need to perform any hashing as `resource.PrefixedUniqueId()` already includes a mutex with a monotonic counter to ensure it is unique even during concurrent operations with the same prefix:

https://github.com/hashicorp/terraform/blob/ad259b2b9569cadd4a24f051e271fa5e669066f2/helper/resource/id.go#L35-L45

We just need to trim the input string 👍

Reply:
That's a lot simpler. Why even bother with a prefix based on the name if it is irrelevant for uniqueness? Why not use "" or "terraform" or "private_dns_namespace"?

Thanks for merging and cleaning up